lib/generators/clearance/install/install_generator.rb

```diff
@@ -126,6 +127,14 @@ def migration_version
           "[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
         end
       end
+
+      def models_inherit_from
+        if Rails.version >= "5.0.0"
```

Comment:
I think both work well, but I'd probably write this as `ActiveRecord::VERSION::MAJOR >= 5`.